as/plugins/org.jboss.tools.as.runtimes.integration/src/org/jboss/tools/as/runtimes/integration/internal/DriverUtility.java

```diff
@@ -71,7 +71,7 @@ public Pair(String loc, int type) {
 		RUNTIME_DRIVER_LOCATION.put(IJBossToolingConstants.EAP_61, new Pair(HSQLDB_DRIVER_72_EAP61_LOCATION, H2_TYPE));
 		RUNTIME_DRIVER_LOCATION.put(IJBossToolingConstants.EAP_70, new Pair(HSQLDB_DRIVER_72_EAP61_LOCATION, H2_TYPE));
 		RUNTIME_DRIVER_LOCATION.put(IJBossToolingConstants.EAP_71, new Pair(HSQLDB_DRIVER_72_EAP61_LOCATION, H2_TYPE));
-		RUNTIME_DRIVER_LOCATION.put(IJBossToolingConstants.WILDFLY_120, new Pair(HSQLDB_DRIVER_72_EAP61_LOCATION, H2_TYPE));
+		RUNTIME_DRIVER_LOCATION.put(IJBossToolingConstants.WILDFLY_130, new Pair(HSQLDB_DRIVER_72_EAP61_LOCATION, H2_TYPE));
```

Comment:
This should be an addition line only, not a deletion and an addition. 
I'll look into merging this after code freeze.